index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once("Autor.php");
+require_once("Libro.php");
+require_once("Revista.php");
+require_once("ImprimirAutor.php");
+
+// Crear autores
+$autor1 = new Autor("Elena White", "EEUU");
```

Comment:
```suggestion
$autorElenaWhite = new Autor("Elena White", "EEUU");
```
Puede usar un nombre más descriptivo